### ISSUE TITLE
error

### ISSUE THREAD

**Issue by kris-wu666** [SamuelMosesA/Doc-Layout-Seg-with-UNET#2]
hello,when I try to run this project , an error has occurred, in dataloaders.py :from .augmentations import RandomFlip, toTensor,this reference dosen't exist

**Comment by SamuelMosesA** [comment]
I may have deleted the file by accident. I have added it now by looking at an older jupyter notebook of it.
However I cannot test it as my laptop is not setup for running pytorch.

**Comment by kris-wu666** [comment]
Absolutely thanks ,I already have tested on my laptop, TRAIN_MASKS_FOLDER, TRAIN_IMAGE_FOLDER, VAL_MASKS_FOLDER,
VAL_IMAGE_FOLDER ,can you add these four folders,they are also missed.

**Comment by SamuelMosesA** [comment]
These folders contained the data. Since the data was very large, I have not added it. 
The dataset used was [Prima Research Layout Analysis Dataset](https://www.primaresearch.org/datasets/Layout_Analysis)
I have added a sample code which I used to create the masks for the dataset.

You can request and download the dataset and use the logic in the sample code to get the segmentation masks

**Comment by kylechang523** [comment]
Same question here， can you provide some sample of the training dataset format? maybe only 10 images?
